README.md

```diff
@@ -84,41 +84,46 @@ The protocol chooses to override at least the following from https://docs.sherlo
 - Discrepancies in handling of zero/non-actionable input amounts (no-op, revert, etc..) are out of scope.
 - Issues in other Makerdao repositories are out of scope, even in case they are relevant to interactions with the contracts in scope. For example - dss, the Chief contract.
 - As the initiation of the endgame components is not considered time-critical, any permissionless action that can cause the spells to revert is not considered a medium or high severity issue (as another spell can be crafted). It can be assumed that no other mission-critical tasks are included in the same spell.
-- Issues where there is damage to the protocol but attack cost exceeds damage caused/funds stolen significantly are considered low severity.
-- Some third parties are trusted - Uniswap, Circle (USDC).
+- Issues where there is damage to the protocol/users but the net attack cost exceeds the damage caused significantly (50%+ more) are considered low severity.
+- Third parties are trusted (for example Uniswap).
 - Potential user errors are out of scope.
 - Tests and mock contracts are excluded.
 - Deployment trust model - deployed contracts are assumed to be checked as part of Maker's spellcrafting process. Deployment of the contracts is assumed to be done with special care taken that all contracts have been deployed correctly. It is assumed that the initcode, bytecode, traces and storage (e.g. mappings) are checked for unintended entries, calls or similar. This is especially crucial for any value stored in a mapping array or similar (e.g. could break access control, could lead to stealing of funds). Additionally, it is checked that no allowance is given to unexpected addresses.
-- Minor rounding errors leading to missing some fees, getting more fee share compared to someone else, or fees locked in the protocol are considered low severity at best.
-- DoS issues where no funds would be locked or protocol functionality is blocked but one one can still withdraw should be considered low severity at best.
-- Oracles are trusted to provide non-stale and correct information.
+- Minor rounding errors leading to missing some fees, getting more fee share compared to someone else, or fees locked in the protocol are considered low severity at best. For definition of "minor" we use 0.5%.   
```

Comment:
Didn't we have to add the minimal values here?

Reply:
done